lang/sr.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"dir": "ltr",
 	"LangCode": "sr",
-	"DateTime": "d.m.yyyy HH:MM",
+	"DateTime": "m/d/yyyy h:MM aa",
```

Comment:
Ah yes, sorry. Anyway, just use whatever format is really used in Serbian/Croatian. If dots are officially used as delimiters, use dots instead of slashes.